pkg/ucauth/kratos/identity.schema.json

```diff
@@ -1,5 +1,5 @@
 {
-  "$id": "https://schemas.ory.sh/presets/kratos/quickstart/email-password/identity.schema.json",
+  "$id": "https://erda.cloud/schemas/identity.schema.json",
```

Comment:
`https://schemas.erda.cloud/kratos/identity.schema.json` is better I think.

Reply:
yes